Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/Person.hs

```diff
@@ -142,16 +127,206 @@ findAllDriversByIdsFirstNameAsc ::
   Id Merchant ->
   [Id Person] ->
   m [FullDriver]
-findAllDriversByIdsFirstNameAsc merchantId driverIds = fmap (map mkFullDriver) $
+findAllDriversByIdsFirstNameAsc merchantId driverIds = do
+  driverLocs <- getDriverLocs driverIds merchantId
+  driverInfos <- getDriverInfos driverLocs
+  vehicle <- getVehicles driverInfos
+  drivers <- getDrivers vehicle
+  return (linkArrays driverLocs driverInfos vehicle drivers)
+
+linkArrays :: [DriverLocation] -> [DriverInformation] -> [Vehicle] -> [Person] -> [FullDriver]
+linkArrays driverLocations driverInformations vehicles persons =
+  let locationHashMap = buildLocationHashMap driverLocations
+      -- personHashMap = buildPersonHashMap persons
```

Comment:
Remove commented out unused code.
Same for everywhere else

Reply:
Done!